test/lib/bridge/content_test.rb

```diff
@@ -105,6 +105,35 @@ class MockBridge
         assert_equal nil, bridge.issue_template_content
       end
     end
+
+    describe 'The content string is not UTF-8' do
+      it 'force encodes an ascii string' do
```

Comment:
I think technically the problem is that the content string _is_ UTF-8, but Ruby is decoding it as ASCII by default to we have to reencode as UTF-8. http://stackoverflow.com/a/16596591 describes what we're seeing pretty well.